client/deposits/index.tsx

```diff
@@ -79,6 +97,33 @@ const NextDepositNotice: React.FC = () => {
 	);
 };
 
+const DepositFailureNotice: React.FC = () => {
+	const { hasErroredExternalAccount } = useAccountStatus();
+
+	return hasErroredExternalAccount ? (
+		<BannerNotice
+			status="warning"
+			icon
+			className="deposit-failure-notice"
+			isDismissible={ false }
+		>
+			{ interpolateComponents( {
+				mixedString: __(
+					'Deposits are currently paused because a recent deposit failed. Please {{updateLink}}update your bank account details{{/updateLink}}.',
+					'woocommerce-payments'
+				),
+				components: {
+					updateLink: (
+						<ExternalLink
+							href={ wcpaySettings.accountStatus.accountLink }
+						/>
+					),
+				},
+			} ) }
+		</BannerNotice>
+	) : null;
```

Comment:
Maybe it can be
```suggestion
	return hasErroredExternalAccount && (
		<BannerNotice
			status="warning"
			icon
			className="deposit-failure-notice"
			isDismissible={ false }
		>
			{ interpolateComponents( {
				mixedString: __(
					'Deposits are currently paused because a recent deposit failed. Please {{updateLink}}update your bank account details{{/updateLink}}.',
					'woocommerce-payments'
				),
				components: {
					updateLink: (
						<ExternalLink
							href={ wcpaySettings.accountStatus.accountLink }
						/>
					),
				},
			} ) }
		</BannerNotice>
	) ;
```